models/models.js

```diff
@@ -8,7 +8,10 @@ exports.selectTopics = () => {
 
 exports.selectArticlesById = (article_id) => {
   return db
-    .query("SELECT * FROM articles WHERE article_id = $1;", [article_id])
+    .query(
+      "SELECT articles.*, (SELECT COUNT(*) FROM comments WHERE comments.article_id = articles.article_id) AS comment_count FROM articles WHERE article_id = $1;",
```

Comment:
I potentially would've done this with a left join instead of nesting a whole other query into that AS statement to save myself making multiple queries but this works too

Reply:
Yes I will do. Thanks for the feedback :)